packages/world/src/Types.sol

```diff
@@ -11,6 +11,12 @@ struct VoxelCoord {
   int32 z;
 }
 
+struct ChunkCoord {
+  int32 x;
+  int32 y;
+  int32 z;
+}
```

Comment:
intentionally creating a different type here to not get confused with the different coordinate systems (blocks vs chunks)